src/menu/menu.scss

```diff
@@ -19,13 +19,14 @@
   margin: 0;
   padding: 0;
 
-  > li {
+  > li:not(.iui-divider) {
```

Comment:
This increases the specificity 🤔

Reply:
Added a `.iui-menu-item` class so I can avoid using this `:not`.

Reply:
The specificity went from 0-1-1 to 0-2-1 to 0-2-0.

If you remove nesting, it will be 0-1-0.